invisible_cities/core/configure.py

```diff
@@ -15,16 +15,16 @@
 from .        log_config      import logger
 from . import system_of_units as     units
 
-from ..types.symbols import EventRange
+from ..types.symbols import ALL_SYMBOLS
+
 
 def event_range(string):
     try:
         return int(string)
     except ValueError:
-        if string.lower() == 'all' : return EventRange.all
-        if string.lower() == 'last': return EventRange.last
-        # TODO Improve the message in the ValueError
-        raise ValueError("`--event-range` must be an int, 'all' or 'last'")
+        if string not in ALL_SYMBOLS:
```

Comment:
I'm unfamiliar with a "not" in this usage.  I would expect to see "if not string in ALL_SYMBOLS:"

Does this provide the same functionality?

Reply:
Yes, and actually it is the recommended syntax in python because it reads more like English.
I think the logic was that `not string in x` could be potentially read as `(not string) in x`. However this explanation clashes with this other syntax: `x is not None`.
`not None` is `True`, but `x is not None` is not the same as `x is True`. For instance for `x=3`
`x is not None` is `True`, but `x is True` is `False`,